library/pathname/relative_path_from_spec.rb

```diff
@@ -0,0 +1,32 @@
+require File.expand_path('../../../spec_helper', __FILE__)
+require 'pathname'
+
+describe "Pathname#relative_path_from" do
+  it "has no common prefix" do
```

Comment:
While I am not very strict on it, usually the concatenated description should be a sentence and describe the behavior:
`Pathname#relative_path_from raises an error when the two paths do not share a common prefix`.
It's definitely some work though, but maybe a few here could be improved for readability? (i.e. it also helps to get a quick idea when a spec fails and when reading a spec for the first time)